openstack/assets/dashboards/openstack-controller.json

```
@@ -0,0 +1,693 @@
+        {
+            "height": 14,
+            "id": 16,
+            "legend": false,
+            "legend_size": "0",
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "sum:openstack.nova.vcpus{$aggregate} by {host}",
+                        "style": {
+                            "palette": "warm"
+                        },
+                        "type": "line"
+                    },
+                    {
+                        "q": "sum:openstack.nova.vcpus_used{$aggregate} by {hypervisor}",
+                        "type": "area"
+                    }
+                ],
+                "viz": "timeseries"
+            },
+            "time": {
+                "live_span": "1h"
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "VCPUs used vs available by hypervisor",
+            "type": "timeseries",
+            "width": 46,
+            "x": 69,
+            "y": 48
+        },
+        {
+            "bgcolor": "gray",
+            "font_size": "36",
+            "height": 7,
+            "html": "[Nova Server](https://www.datadoghq.com/blog/openstack-monitoring-nova/#nova-server-metrics)",
+            "id": 17,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "bottom",
+            "tick_pos": "50%",
+            "type": "note",
+            "width": 46,
+            "x": 117,
+            "y": 66
+        },
+        {
+            "height": 14,
+            "id": 18,
+            "legend": false,
+            "legend_size": "0",
+            "tile_def": {
+                "requests": [
+                    {
+                        "q": "avg:openstack.nova.server.hdd_read_req{$project} by {host}",
+                        "style": {
+                            "palette": "dog_classic",
+                            "type": "solid",
+                            "width": "normal"
+                        },
+                        "type": "area"
+                    }
+                ],
+                "viz": "timeseries",
+                "yaxis": {
+                    "includeZero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "time": {
+                "live_span": "1h"
+            },
+            "title": true,
+            "title_align": "left",
+            "title_size": 16,
+            "title_text": "HDD read rate by instance for $project",
+            "type": "timeseries",
+            "width": 46,
+            "x": 117,
+            "y": 75
+        },
+        {
+            "height": 10,
+            "id": 19,
+            "sizing": "fit",
+            "type": "image",
+            "url": "https://app.datadoghq.com/static/images/saas_logos/bot/openstack.png",
```

```suggestion
            "url": "https://app.datadoghq.com/static/images/saas_logos/bot/openstack@2x.png",
```

Higher resolution :)